src/convertToObject.js

```diff
@@ -5,8 +5,27 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter((declaration) => declaration !== '')
+    .map((declaration) => {
+      const propertyValuePair = declaration.split(':');
+
+      propertyValuePair[0] = propertyValuePair[0].trim();
```

Comment:
Directly modifying the array elements within `map` callback (like `propertyValuePair[0] = propertyValuePair[0].trim()`) is not recommended as `map` should be used to return a new value rather than modifying the existing one. Instead, you can return a new array with trimmed values.

Reply:
it doesn't matter if I do a two colon check as said at the bottom